### DISCUSSION TITLE
Invalid template name in 'extends' tag when viewing interface tab

### DISCUSSION THREAD

**Discussion by gizmo15** [netbox-community/netbox#13572] [Help Wanted!]
Hi,

I have this error when i wan't to see/after add an interface :
```
<class 'django.template.exceptions.TemplateSyntaxError'>

Invalid template name in 'extends' tag: ''. Got this from the 'base_template' variable.

Python version: 3.9.17
NetBox version: 3.5.7
Plugins: None installed
```

But when i add an interface, the interface is added correctly but i can't see the interfaces tab.

I have a custom field : vm id and a custom tag : tools_provisionning

I can see all tabs except interfaces.

Can you help me on that ?

Thanks!

**Comment by abhi1693** [comment]
This is probably due to https://github.com/netbox-community/netbox/compare/v3.5.7...v3.5.8#diff-0e1fb65d1356bc6621fc0ce8d2d9455dba5ca6febd5652adc913afef78e23e22R146

Could you give more detailed steps to reproduce this issue?

**Comment by gizmo15** [comment]
i go on virtualization/virtual machines/a virtual VM and goto interfaces tab.
Same error for a new vm or an older one.

But, if i click on add interface when i on the general tab of a virtual machine, i can follow the form but when i click on create, i have the same error but the interface is well created.
I can attribute an ip address to this interface without any problem.

If i create a virtual machine without any tag or custom field, same error

**Reply by abhi1693** [reply]
I do not have this issue on v3.5.8 

Existing VM
![image](https://github.com/netbox-community/netbox/assets/5083532/178cecfe-d66b-48c3-80a5-38c85edea88c)

New VM
![image](https://github.com/netbox-community/netbox/assets/5083532/4367dc4b-f12a-4eb9-985f-a6dfe63100d1)

**Comment by abhi1693** [comment]
Did you restart netbox services after an upgrade? Because it may be that your instance is not able to see the new templates that we shipped in v3.5.7

**Comment by gizmo15** [comment]
that was that! a process was stuck, a full stop/start and it's ok.

Thanks for your help!
